main.js

```diff
@@ -0,0 +1,155 @@
+const API = "27512711";
```

Comment:
https://user-images.githubusercontent.com/34886027/204142369-5a630ad2-38fa-45a9-ac3b-dce4767b2662.mov

<img width="609" alt="Screenshot 2022-11-27 at 17 00 49" src="https://user-images.githubusercontent.com/34886027/204142373-16f29913-aefb-49c3-ad0b-bdf093493e07.png">

<img width="95" alt="Screenshot 2022-11-27 at 17 01 13" src="https://user-images.githubusercontent.com/34886027/204142377-7edf6f6d-c38a-4f4b-ab41-c3b421cf31d3.png">

<img width="1272" alt="Screenshot 2022-11-27 at 17 04 29" src="https://user-images.githubusercontent.com/34886027/204142379-848312e2-3cea-4982-83d6-6af5ada05d2b.png">

Reply:
UI is a bit "broken". Also 3 API requests on each pagination call 🤔